reader/lib/migrations/index.js

```diff
@@ -0,0 +1,58 @@
+import fs from 'fs';
+import { fileURLToPath } from 'url';
+import path from 'path';
+import { uniq, get } from 'lodash'
+import minimist from 'minimist';
+import cache from './cache.json';
+import config from './config';
+
+const argv = minimist(process.argv.slice(2));
```

Comment:
Given we already have [commander](https://github.com/nhsx/standards-registry/pull/318/files#diff-6a3623295cfb788b8cf298e4430aca04ab92eeb81d860dcb01be176c83b2d106L14) maybe it's worth replacing this with 
`commander.parse(process.argv)`